graylog2-web-interface/src/views/components/messagelist/MessageFields/MessageFields.tsx

```diff
@@ -14,48 +14,98 @@
  * along with this program. If not, see
  * <http://www.mongodb.com/licensing/server-side-public-license>.
  */
-import * as React from 'react';
+
+import React, { useContext, useState, useCallback } from 'react';
 import styled, { css } from 'styled-components';
 
-import { MessageDetailsDefinitionList } from 'components/common';
-import MessageField from 'views/components/messagelist/MessageField';
-import FieldType from 'views/logic/fieldtypes/FieldType';
-import FieldTypeMapping from 'views/logic/fieldtypes/FieldTypeMapping';
-import useFeature from 'hooks/useFeature';
-import MessageFieldsViewMode from 'views/components/messagelist/MessageFields/MessageFieldsViewMode';
-import type { MessageFieldsComponentProps } from 'views/components/messagelist/MessageFields/types';
+import MessageFieldsViewModeList from 'views/components/messagelist/MessageFields/MessageFieldsViewModeList';
+import useFormattedFields from 'views/components/messagelist/MessageFields/hooks/useFormattedFields';
+import MessageFavoriteFieldsContext from 'views/components/contexts/MessageFavoriteFieldsContext';
+import { Icon } from 'components/common';
+import Store from 'logic/local-storage/Store';
+import useSendTelemetry from 'logic/telemetry/useSendTelemetry';
+import { TELEMETRY_EVENT_TYPE } from 'logic/telemetry/Constants';
 
-export const MessageDetailsDL = styled(MessageDetailsDefinitionList)(
+const Line = styled.div(
   ({ theme }) => css`
-    color: ${theme.colors.text.primary};
-
-    dd {
-      &:not(:last-child) {
-        border-bottom: 1px solid ${theme.colors.gray[90]};
-      }
-    }
+    border-top: 1px ${theme.colors.table.row.divider} solid;
+    flex-grow: 1;
+    height: 1px;
   `,
 );
 
-export const DefaultMessageFields = ({ message, fields }: MessageFieldsComponentProps) => {
-  const formattedFields = message.formatted_fields;
-  const renderedFields = Object.keys(formattedFields)
-    .sort()
-    .map((key) => {
-      const { type } = fields.find((t) => t.name === key, undefined, FieldTypeMapping.create(key, FieldType.Unknown));
+const SeparatorContainer = styled.div(
+  ({ theme }) => css`
+    display: flex;
+    align-items: center;
+    color: ${theme.colors.table.row.divider};
```

Comment:
The separator is almost invisible in the dark theme:

<img width="2405" height="691" alt="Image" src="https://github.com/user-attachments/assets/ef4516dd-48ff-40ec-ab62-5df7960a9535" />

Reply:
<img width="281" height="90" alt="Screenshot 2025-11-12 at 20 31 03" src="https://github.com/user-attachments/assets/40dd9169-0997-46c9-9099-0aa95458e29a" />
<img width="291" height="91" alt="Screenshot 2025-11-12 at 20 31 18" src="https://github.com/user-attachments/assets/fa530bde-ff3f-48c8-b170-d60bfaa72f0c" />
Add other color for text